provision/aws/rosa_cluster_reaper.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+set -e
+
+if [ -f ./.env ]; then
+  source ./.env
+fi
+
+echo "Starting the ROSA cluster reaper at $(date -uIs)"
+
+# Get a list of running ROSA clusters
+CLUSTERS=$(rosa list clusters --output json | jq -r '.[] | select(.state == "ready") | .name')
+
+if [ -z "$CLUSTERS" ]; then echo "Didn't find any ready state clusters"; exit 1; fi
+
+# Loop through each cluster
+for CLUSTER_NAME in $CLUSTERS; do
+  echo "Checking Cluster: $CLUSTER_NAME"
+  ./rosa_oc_login.sh
+  #Check if the 'keepalive' namespace exists in the specific cluster
+  keepalive=$(oc get projects | grep "keepalive")
```

Comment:
Do you think it would make sense to introduce some script that would create the `keepalive` namespace, so we avoid some problems with spelling or something similar?

Also, from other parts of kcb I had a feeling we are trying to use `kubectl` where possible. Should we change this too?

Reply:
for keepalive, I can add a small task command or bash script to do it that way., or have an option during the rosa_cluster_create.sh bash script itself to have a flag to create that empty namespace. Alternatively, we can add the below snippet to docs., choice would be ours. Let me know which you think would work best.

```
oc new-project hello-openshift \
--description="This is an example project" \
--display-name="Hello OpenShift"
```

On your second question, I was leading off of the `rosa_oc_login.sh` script, where indeed use `oc` to login to the cluster, wouldn't it be confusing to use `kubectl` after that ?

Reply:
I think we can do both. task command sounds good to me for local usage. Not sure what you mean by docs, but I guess we should add some info about this to https://github.com/keycloak/keycloak-benchmark/blob/main/.github/workflows/README.md

Reply:
Regarding `kubectl`, I don't have any strong opinion on that. I am just pointing out that in other places we usually use `kubectl`.

Reply:
> I think we can do both. task command sounds good to me for local usage. Not sure what you mean by docs, but I guess we should add some info about this to https://github.com/keycloak/keycloak-benchmark/blob/main/.github/workflows/README.md

Perfect, will implement that and add it to the README as well. Thanks @mhajas

Reply:
applied the changes now. @mhajas when you have a chance pls review again